src/Common/src/Interop/Unix/System.Native/Interop.SchedGetSetAffinity.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        internal unsafe struct CpuSetBits
```

Comment:
I'll change the field to be private so that it's still visible what the struct contains and how it fits with the native implementation.